modules/parsers/parser-hana/src/main/java/exceptions/TableFunctionMissingPropertyException.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2022 SAP SE or an SAP affiliate company and XSK contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and XSK contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package exceptions;
```

Comment:
I really don't like organising packages by layers instead of features, however I know that this is the current approach especially in the parsers, but can we move this exception in the package where it is used and remove this "exceptions" package?